content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -12,6 +12,7 @@ weight: 10
 
 ## Security
 
+- **Added** the [`v1beta1` authorization policy](/docs/concepts/security#authorization) for enforcing access control. This deprecates the [`v1alpha1` RBAC policy](/docs/reference/config/authorization/istio.rbac.v1alpha1/).
```

Comment:
```suggestion
- **Added** the [`v1beta1` authorization policy model](/docs/concepts/security#authorization) for enforcing access control. This will eventually replace the [`v1alpha1` RBAC policy](/docs/reference/config/authorization/istio.rbac.v1alpha1/).
```